src/v/datalake/coordinator/iceberg_file_committer.cc

```diff
@@ -384,6 +385,42 @@ iceberg_file_committer::commit_topic_files_to_catalog(
     }
     auto& main_table_commit_builder = main_table_commit_builder_res.value();
 
+    // DLQ table (optional):
+    std::optional<table_commit_builder> dlq_table_commit_builder;
+    {
+        // TODO(iceberg-dlq): Load it on demand.
+        //  For now, we load it unconditionally because once we start processing
+        //  pending entries we are not allowed to suspend/co_await.
```

Comment:
nit: spacing's a little off

Reply:
It was intention here to show that the paragraph belongs to the TODO above.